CHANGELOG.md

```diff
@@ -1,5 +1,61 @@
 # CHANGELOG
 
+## 1.28.0 (2022-08-09)
+
+MLflow 1.28.0 includes several major features and improvements:
+
+Features:
+
+- [Pipelines] Log the full Pipeline runtime configuration to MLflow Tracking during Pipeline execution (#6359, @jinzhang21)
+- [Pipelines] Add ``pipeline.yaml`` configurations to specify the Model Registry backend used for model registration (#6284, @sunishsheth2009)
+- [Pipelines] Support optionally skipping the ``transform`` step of the scikit-learn regression pipeline (#6362, @sunishsheth2009)
+- [Pipelines] Add UI links to Runs and Models in Pipeline Step Cards on Databricks (#6294, @dbczumar)
+- [Tracking] Introduce ``mlflow.search_experiments()`` API for searching experiments by name and by tags (#6333, @WeichenXu123; #6227, #6172, #6154, @harupy)
+- [Tracking] Increase the maximum parameter value length supported by File and SQL backends to 500 characters (#6358, @johnyNJ)
+- [Tracking] Introduce an ``--older-than`` flag to ``mlflow gc`` for removing runs based on deletion time (#6354, @Jason-CKY)
+- [Tracking] Add ``MLFLOW_SQLALCHEMYSTORE_POOL_RECYCLE`` environment variable for recycling SQLAlchemy connections (#6344, @postrational)
+- [UI] Display deeply nested runs in the Runs Table on the Experiment Page (#6065, @tospe)
+- [UI] Add box plot visualization for metrics to the Compare Runs page (#6308, @ahlag)
+- [UI] Display tags on the Compare Runs page (#6164, @CaioCavalcanti)
+- [UI] Use scientific notation for axes when viewing metric plots in log scale (#6176, @RajezMariner)
+- [UI] Add button to Metrics page for downloading metrics as CSV (#6048, @rafaelvp-db)
+- [UI] Include NaN and +/- infinity values in plots on the Metrics page (#6422, @hubertzub-db)
+- [Tracking / Model Registry] Introduce environment variables to control retry behavior and timeouts for REST API requests (#5745, @peterdhansen)
```

Comment:
Shall we allow whitespace in tag name ? Or use `Model-Registry` instead ?

Reply:
We use "Model Registry" in the changelog :)